functions/getCoords/source.js

```diff
@@ -60,5 +58,14 @@ exports = async function(address){
   }
   let coords = responseJson.results[0].geometry.location;
```

Comment:
I think you need to set the coords to results[x] depending on which index has a zip code that matches the users input. maybe we can talk about this when you have a sec.

Reply:
From my tests so far, it looks like we only care about the data in results[0]. In line 61  of my changes I look for the zipcode in results[0].